content/03.methods.md

```diff
@@ -64,14 +64,26 @@ Alignments were futher processed using following the Broad Institute's Best Prac
 Duplicates were marked using Samblaster[@doi:10/f6kft3] v0.1.24, BAMs merged and sorted using Sambamba [@doi:10/gfzsfw] v0.6.3.
 Lastly, resultant BAMs were processing using Broad's Genome Analysis Tool Kit (GATK) [@url:https://software.broadinstitute.org/gatk/] v4.0.3.0, BaseRecalibrator submodule.
 
-### Germ Line Single Nucleotide Variant Calling
+### Quality Control of Sequencing Data
+NGSCheckmate [doi:10.1093/nar/gkx193] was peformed on matched tumor/normal crams to confirm sample matches and remove mis-matched samples from the dataset. 
+Cram inputs were preprocessed using bcftools to filter and call snps using default parameters[@url:https://github.com/parklab/NGSCheckMate] and the resulting VCFs were used to run NGSCheckmate using [this workflow](https://github.com/d3b-center/ngs_checkmate_wf) in the D3b GitHub repository. 
+Per author guidelines, <= 0.61 was used as a correlation coefficient cutoff at sequencing depths >10 to predict mismatched samples.
 
 ### Somatic Single Nucleotide Variant Calling
 #### SNV and INDEL calling
 
-We used Strelka2 [@doi:10/gdwrp4] v2.9.3 and Mutect2 from GATK v4.1.1.0.
-Strelka2 was run using default parameters on human genome reference hg38, canonical chromosomes only (chr1-22, X,Y,M), as recommended by the author.
-Mutect2 was run following Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl].  
+We used four variant callers to call SNVs and INDELS from Panel, WXS, and WGS data: Strelka2, Mutect2, Lancet, and Vardict.
+The same input interval BED files were used for both panel and WXS data.
+Strelka2 [@doi:10/gdwrp4] v2.9.3 was run using default parameters on human genome reference hg38, canonical chromosomes only (chr1-22, X,Y,M), as recommended by the authors.
+The final Strelka2 VCF was filtered for PASS variants.
+Mutect2 from GATK v4.1.1.0 was run following Broad best practices outlined from their Workflow Description Language (WDL) [@url:https://github.com/broadinstitute/gatk/blob/4.1.1.0/scripts/mutect2_wdl/mutect2.wdl].
+The final Mutect2 VCF was filtered for PASS variants. 
+Lancet [@doi:10.1038/s42003-018-0023-9] v1.0.7 [@url:https://github.com/nygenome/lancet].
```

Comment:
I don't think this is a sentence. Should it be?

Reply:
haha, nope! will fix this.